README.md

```diff
@@ -50,13 +50,13 @@ to keep track of for which architectures the weights are valid for.
 
 See [examples/digits.jl](examples/digits.jl) for a larger example.
 
-## `LegolasFlux.ModelRow`
+## `LegolasFlux.ModelV1`
```

Comment:
ah yeah, even though this isn't exported I think it's probably safer to tag this as breaking.

Reply:
We could continue to keep the alias, but I am afraid of having no backport version #s